ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,17 +48,16 @@ class FaultInjector
     VelocityLimit=7, 
     NoForceData=8};
 
-  enum SystemFaults {
-    System=1, 
-    ArmGoalError=2, 
-    ArmExecutionError=4,
-    TaskGoalError=8, 
-    CamGoalError=16, 
-    CamExecutionError=32, 
-    PtGoalError=64, 
-    PtExecutionError=128, 
-    LanderExecutionError = 256};
-
+	static constexpr std::bitset<9> isSystem{		0b0'0000'0001 };
+	static constexpr std::bitset<9> isArmGoalError{		0b0'0000'0010 };
+	static constexpr std::bitset<9> isArmExecutionError{		0b0'0000'0100 };
+	static constexpr std::bitset<9> isTaskGoalError{	0b0'000'1000 };
+	static constexpr std::bitset<9> isCamGoalError{	0b0'0001'0000 };
+	static constexpr std::bitset<9> isCamExecutionError{	0b0'0010'0000 };
+	static constexpr std::bitset<9> isPtGoalError{		0b0'0100'0000 };
+	static constexpr std::bitset<9> isPtExecutionError{	0b0'1000'0000 };
```

Comment:
I'm blanking on "Pt" could mean.  Spelling it out would be helpful.

Reply:
I believe it's Pan Tilt? I can add that in

Reply:
That makes sense, yes please  change to PanTilt, since we need antenna faults.